lib/autokey/scripting/engine.py

```diff
@@ -132,7 +132,7 @@ def create_phrase(self, folder, name: str, contents: str,
                       hotkey: Tuple[List[Union[model.Key, str]], Union[model.Key, str]]=None,
                       send_mode: model.SendMode=model.SendMode.KEYBOARD, window_filter: str=None,
                       show_in_system_tray: bool=False, always_prompt: bool=False,
-                      temporary=False):
+                      temporary=False, replaceExistingHotkey=False):
```

Comment:
Python PEP8 (code style recommendation) says to not use camelCase for variables and parameters. instead use lower_case_with_underscores. Adding CamelCase exactly _here_ even mixes both styles in a public API function call.

Reply:
Oops, missed that. Sorry